org.eclipse.jdt.core/model/org/eclipse/jdt/internal/core/builder/ClasspathJrtWithReleaseOption.java

```diff
@@ -137,16 +134,16 @@ Map<String, SimpleSet> findPackagesInModules() {
 		if (this.modPathString == null) {
 			return Map.of();
 		}
-		Map<String, SimpleSet> cache = PackageCache.computeIfAbsent(this.modPathString, key -> {
+		return PackageCache.computeIfAbsent(this.modPathString, key -> {
```

Comment:
Please keep such code "as is", it's otherwise hard to debug maps hits etc.

Reply:
Revered to previous state.